### DISCUSSION TITLE
How to use 'purgePendingUsersDuration'?

### DISCUSSION THREAD

**Discussion by carolineboeykens** [craftcms/cms#13384] [Q&A] [answered]
We added this option to the config/general.php

```
 'purgePendingUsersDuration' => 5270400, // 61 days
```

But after running `./craft gc/run` it does not seem to be deleting older pending users.
Are we using this wrong? How can we implement this?

**Comment by brandonkelly** [comment, marked as answer]
Only users who have been sitting on an activation link (e.g. sent to their email) for longer than the `purgePendingUsersDuration` setting will be deleted. Do the users you want to delete have `verificationCodeIssuedDate` values?

**Reply by carolineboeykens** [reply]
The users in our system have no password or login, so that was never gonna work.
It sounds logic now!
Thank you for the explanation!
